lionweb/src/main/kotlin/com/strumenta/kolasu/lionweb/StarLasuLWLanguage.kt

```diff
@@ -23,6 +28,16 @@ object StarLasuLWLanguage : Language("com.strumenta.StarLasu") {
         astNode.addReference("transpiledNode", astNode, Multiplicity.MANY)
 
         addPlaceholderNodeAnnotation(astNode)
+
```

Comment:
Would it make sense to update the language version?

Reply:
I don't know the implications, do you think we should update it?

Reply:
The implications should be minimal, provided we do not try to load models serialized with a previous version. That would probably cause issues. Probably, given we are just adding elements and not doing other changes, we can just keep the current version, and reserve version updates for breaking changes. I will close this comment. Please feel free to merge the PR